content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -184,6 +184,16 @@ Used on: PersistentVolumeClaim
 
 This annotation has been deprecated.
 
+### volume.beta.kubernetes.io/mount-options (deprecated) {#mount-options}
+
+Example : `volume.beta.kubernetes.io/mount-options: ["ro", "soft"]`
```

Comment:
This doesn't look right: annotations can only have a string value.

Maybe this:
```suggestion
Example : `volume.beta.kubernetes.io/mount-options: "ro,soft"`
```